### DISCUSSION TITLE
What's next?

### DISCUSSION THREAD

**Discussion by hekopath** [cyberman54/ESP32-Paxcounter#710] [General]
First of all, my special appreciation for the work that goes into this project.

In the time I've been following this project, I've observed two things. On the one hand, the code for the actual function is being revised and improved.
On the other hand, there are always requests to integrate certain sensors.
For the first point there are the classic Github tools and methods based on issues and PRs.

In the second case, the discussion area would possibly be a place for interested users to exchange ideas outside the source code and, if necessary, to provide assistance (for example, such as the "custom sensor" placeholders can be used).
If certain quality criteria are met and a certain interest from several users can be identified, this can result in an "issue" or "feature request" with a well designed place in the project.

The intention behind this idea is to relieve the safe programmers and still offer the interested user a platform.

**Comment by hekopath** [comment]
I'm just realizing that the "Welcome" discussion topic has the same content with far fewer words.

**Reply by FlorianLudwig** [reply]
Hi @hekopath maybe you could transform your thoughts into actionable ideas and add them to the welcome thread?

And we mark this one as done?

**Comment by hekopath** [comment]
The purpose of the discussion was to encourage others to express an opinion. That it was formulated relatively vaguely was more on purpose.
As it turns out, nobody wants to comment on his further thoughts.
Therefore, this discussion can of course also be closed.
